app/controllers/medicaid/income_job_number_continued_controller.rb

```diff
@@ -5,11 +5,17 @@ class IncomeJobNumberContinuedController < MedicaidStepsController
     private
 
     def skip?
-      not_employed? || current_application.number_of_jobs < 4
+      multi_member_household? ||
+        not_employed? ||
+        current_application.number_of_jobs < 4
     end
 
     def not_employed?
       !current_application.employed?
     end
+
+    def multi_member_household?
```

Comment:
I've seen in our current code that there are places where single_member_household is defined as `current_application.members.empty? || current_application.members.count == 1`. I'm not sure how we would get an application without any members (maybe only in tests?), but am thinking we should standardize on single member as either 1 member or 0-1 members.

Reply:
yeah I agree! maybe we decide on one set of logic put the method in `ApplicationController`?

Reply:
what do you think: should we go for 1 or 0?

Reply:
set to `== 1` and moved to paren controller at b0005974